packages/design-system/package.json

```diff
@@ -0,0 +1,153 @@
+  },
+  "typings": "src/interfaces/index.d.ts",
+  "main": "dist/design-system.cjs.js",
+  "module": "dist/design-system.es.js",
+  "jsnext:main": "dist/design-system.es.js",
+  "style": "dist/design-system.css",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "start": "rollup -c --watch",
+    "build": "rollup -c",
+    "prebuild": "rimraf dist",
+    "build:cdn": "npm run build && mkdir -p dist/cdn && cp dist/design-system.css dist/cdn/styles.css && cp dist/cdn/styles.css dist/cdn/styles-v$npm_package_version.css",
+    "prepare": "npm run build",
+    "test": "jest",
+    "test:update": "jest -u",
+    "test:watch": "jest --watchAll",
+    "test:coverage": "jest --coverage",
+    "eslint": "eslint --ext .js,.ts src/ ",
+    "eslint:fix": "eslint --ext .js,.ts src/ --fix "
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/prop-types": "^15.5.6",
+    "@types/react": "^16.4.15",
+    "@types/react-dom": "^16.0.8",
+    "autoprefixer": "^9.0.1",
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^8.2.6",
+    "babel-jest": "^23.4.2",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-css-modules-transform": "^1.6.1",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-react-svg": "^2.1.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-imports": "^1.5.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.15",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-es2015-rollup": "^3.0.0",
+    "babel-preset-react": "^6.3.13",
+    "caniuse-lite": "1.0.30000974",
+    "cross-env": "^5.2.0",
+    "css-loader": "^2.1.1",
+    "cssnano": "^4.0.5",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.4",
+    "eslint": "^4.19.1",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-import-resolver-webpack": "^0.10.0",
+    "eslint-plugin-babel": "^5.1.0",
+    "eslint-plugin-import": "^2.12.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-prettier": "^2.6.2",
+    "eslint-plugin-react": "^7.8.2",
+    "eslint-plugin-typescript": "^0.12.0",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "gh-pages": "^1.2.0",
+    "husky": "^1.3.1",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^23.4.2",
+    "lint-staged": "^8.1.5",
+    "mini-css-extract-plugin": "^0.5.0",
+    "node-eval": "^2.0.0",
+    "node-sass": "^4.9.3",
+    "normalize.css": "^8.0.0",
+    "postcss-flexbugs-fixes": "^4.0.0",
+    "postcss-global-import": "^1.0.6",
+    "postcss-import": "^11.1.0",
+    "postcss-load-config": "^2.0.0",
+    "postcss-loader": "^2.1.6",
+    "postcss-modules": "^1.3.2",
+    "postcss-url": "^7.3.2",
+    "precss": "^3.1.2",
+    "prettier": "1.14.2",
```

Comment:
similarly prettier is a monorepo-level tool, so it shouldnt be kept here

Reply:
done at 860e6b7d0907e3e55330982f896d15ba70f95cc5